src/Beacon/BeaconDemo/Properties/AssemblyInfo.cs

```diff
@@ -2,35 +2,10 @@
 using System.Runtime.CompilerServices;
```

Comment:
All these `AssemblyInfo.cs` files can be deleted. These projects aren't using COM.